server/storage.lua

```diff
@@ -54,10 +54,16 @@ function FetchVehicleEntityByPlate(plate)
     return MySQL.single.await('SELECT * FROM player_vehicles WHERE plate = ?', {plate})
 end
 
----@param plate string
+---@param vehicleId integer
+---@return VehicleEntity
+function FetchVehicleEntityByVehicleId(vehicleId)
+    return MySQL.single.await('SELECT * FROM player_vehicles WHERE vehicleId = ?', {vehicleId})
+end
```

Comment:
A new function is created but not used?

Reply:
Yeah, as a future replacement for `FetchVehicleEntityByPlate` which I guess I should mark as `depracated` maybe?

Reply:
I think this is a situation where we can just change the callers with the function as well. This isn't an API in which we need to worry about breaking compatibility.

Reply:
Fixed in server/main and in here